app/shared/storage.js

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
We should be careful about using node dependencies in `shared/` as the UI can pull these in which is breaking the assumption that the frontend can be loaded in a web context.  Some of this should be fixed through our build tools (i.e. the frontend should be packaged up as a web site).  But we should also think about the directory structure and how it related to this (i.e. should ui/ be in a different top-level directory, or should main/services/shared all be moved down one directory from here?).  Bigger than this issue, but we should discuss (cc: @jsantell)